cfgov/unprocessed/apps/owning-a-home/config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "rateCheckerAPI": "/oah-api/rates/",
-  "countyAPI":      "/oah-api/county/"
+  "rateCheckerAPI": "/oah-api/rates/rate-checker",
+  "countyAPI":      "/oah-api/county/rater-checker"
```

Comment:
is rater-checker correct?

Reply:
lol, I can't even copy \ paste today

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3901/commits/17aadeff078baec96bd7cbd4ec93c9f27b4cc95e